src/coreclr/vm/method.hpp

```diff
@@ -1817,6 +1817,11 @@ class MethodDesc
         LIMITED_METHOD_CONTRACT;
         VolatileStore(&m_interpreterCode, interpreterCode);
     }
+    void ClearInterpreterCodePointer()
+    {
+        LIMITED_METHOD_CONTRACT;
+        VolatileStore(&m_interpreterCode, dac_cast<PTR_InterpByteCodeStart>((TADDR)NULL));
```

Comment:
Will this blow away your new poisoning mechanism from the other PR?

Reply:
Yes. But that's intentional. The reset mechanism makes the MethodDesc back to its initial state ready for being associated with other code.